connmgr.go

```diff
@@ -286,10 +286,8 @@ func (cm *BasicConnMgr) trim() {
 	}
 
 	// do the actual trim.
-	defer log.EventBegin(cm.ctx, "connCleanup").Done()
```

Comment:
Not 100% sure if this is ok.

Reply:
Eh, that's fine.

Reply:
We'll need to do a tracing pass, but that will be an entire project.